From d7ee4a3bed611a0fbd755774b5b9c827b68cbd64 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 4 Jun 2014 07:11:39 +0200
Subject: [PATCH 152/509] spi: zynq-qspi: Make of_device_id array const

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit f8f455351636ca30a5a092a943de3c313c6e8d07

Make of_device_id array const, because all OF functions handle
it as const.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 drivers/spi/spi-zynq-qspi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/spi/spi-zynq-qspi.c b/drivers/spi/spi-zynq-qspi.c
index 4764c20..e109212 100644
--- a/drivers/spi/spi-zynq-qspi.c
+++ b/drivers/spi/spi-zynq-qspi.c
@@ -817,7 +817,7 @@ static int zynq_qspi_remove(struct platform_device *pdev)
 	return 0;
 }
 
-static struct of_device_id zynq_qspi_of_match[] = {
+static const struct of_device_id zynq_qspi_of_match[] = {
 	{ .compatible = "xlnx,zynq-qspi-1.0", },
 	{ /* end of table */ }
 };
-- 
1.7.5.4

